### DISCUSSION TITLE
Unable to run fs.pickle within Cowrie

### DISCUSSION THREAD

**Discussion by gwbobjosh** [telekom-security/tpotce#1196] [Q&A]
Hello,

I am trying to create a custom file system with Cowrie.  I run "bin/fsctl share/cowrie/fs.pickle" and the server returns " env: can't execute 'python': No such file or directory."  It seems like the pointer to Python isn't working.  I have rebuilt the server a number of time incase something weird happened during the install.  I setup a new server and just installed Cowrie, not T-POT or anything else and i was able to open the fs.pickle with no problem.  Seems like something within the Corwie docker is messed up but i have no idea how to fix it.  Anyone else run into this issue?

**Comment by t3chn0m4g3** [comment]
`fsctl` expects `python` which is not automatically linked in Alpine. I added a link to `python3` and deployed a new docker image. However changes will not be stored since the container is started as read_only. Adjust the `/opt/tpot/etc/tpot.yml` to your needs and check the Wiki to work with volumes.
